xgi/drawing/draw.py

```diff
@@ -1578,83 +1589,116 @@ def draw_dihypergraph(
+        Only used if the higher-order network is a `DiHypergraph`.
     rescale_sizes: bool, optional
         If True, linearly interpolate `node_size` and between min/max values
         that can be changed in the other argument `params`.
         If `node_size` is a single value, this is ignored. By default, True.
-    iterations : int, optional
-        Maximum number of iterations taken to recompute the layout.
-        An original partial layout is computed solely based on the original nodes.
-        A full initial layout is then computed by simply adding the "edge" nodes
-        at the barycenters. This initial layout may suffer from overlap between edge-nodes.
-        After that, a spring layout is ran starting from the
-        initial layout, and each iteration makes all of the nodes overlap less.
+    aspect : {"auto", "equal"} or float, optional
+        Set the aspect ratio of the axes scaling, i.e. y/x-scale. `aspect` is passed
+        directly to matplotlib's `ax.set_aspect()`. Default is `equal`. See full
+        Set the aspect ratio of the axes scaling, i.e. y/x-scale. `aspect` is passed
+        directly to matplotlib's `ax.set_aspect()`. Default is `equal`. See full
+        description at
+        https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.set_aspect.html
     **kwargs : optional args
         Alternate default values. Values that can be overwritten are the following:
-        * "min_node_size" (default: 5)
-        * "max_node_size" (default: 30)
-        * "min_node_lw" (default: 0)
-        * "max_node_lw" (default: 5)
-        * "min_lines_lw" (default: 0)
-        * "max_lines_lw" (default: 50)
-        * "min_source_margin" (default: 0)
-        * "min_target_margin" (default: 0)
+        * min_node_size (default: 5)
+        * max_node_size (default: 30)
+        * min_node_lw (default: 0)
+        * max_node_lw (default: 5)
+        * min_edge_marker_size (default: 5)
+        * max_edge_marker_size (default: 30)
+        * min_edge_marker_lw (default: 0)
+        * max_edge_marker_lw (default: 5)
+        * min_dyad_lw (default: 1)
+        * max_dyad_lw (default: 10)
+        * min_source_margin (default: 0)
+        * min_target_margin (default: 0)
+
 
     Returns
     -------
-    ax : matplotlib Axes
-        Axes plotted on
+    ax : matplotlib.pyplot.axes
+        The axes corresponding the visualization
     collections : a tuple of 3 collections:
         * node_collection : matplotlib PathCollection
             Collection containing the nodes
-        * phantom_node_collection : matplotlib PathCollection
-            Collection containing the nodes representing edges
+        * edge_marker_collection : matplotlib PathCollection
+            Collection containing the edge markers
+        * dyad_collection : matplotlib LineCollection if undirected,
+                            list of FancyArrowPatches if not
+            Collection containing the edges
 
     Raises
     ------
```

Comment:
Below:

If something different than a Hypergraph is passed.

Reply:
Fixed.